vng_api_common/apps.py

```diff
@@ -58,3 +59,9 @@ def register_serializer_field():
 
     if RelativeDeltaField is not None:
         mapping[RelativeDeltaField] = DurationField
+
+
+def set_custom_hyperlinkedmodelserializer_field():
```

Comment:
please add a comment that this monkey-patches DRF to avoid having to change all the base classes in existing code